input/docs/handbook/commands/asynchronous-operations.md

```diff
@@ -1,127 +1,134 @@
-# Asynchronous operations
+### Creating commands
```

Comment:
We should probably rename the file since it's more about creating commands now rather than just async.